lib/layout.dart

```diff
@@ -4,16 +4,16 @@ import 'package:soundspace/widgets/large_screen.dart';
 import 'package:soundspace/widgets/side_menu.dart';
 import 'package:soundspace/widgets/small_screen.dart';
 
-
 class SiteLayout extends StatelessWidget {
+  SiteLayout({Key? key}) : super(key: key);
   final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey();
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       key: scaffoldKey,
-      drawer: Drawer(child: SideMenu()),
-      body: ResponsiveWidget(
+      drawer: const Drawer(child: SideMenu()),
```

Comment:
I dis like very much this, specifically b/c const tends to cause more problems than it is worth in dart.

Reply:
This is what the linter wants. I think the actions taken within the widget are still non-const, it's just the presence of the widget in this section